src/editor.cpp

```diff
@@ -0,0 +1,580 @@
+#include "editor.hpp"
+
+#include "exceptions.hpp"
+#include "logger.hpp"
+#include "utils.hpp"
+
+#include "vpdecc/vpdecc.h"
+
+namespace vpd
+{
+using namespace types;
+using namespace constants;
+
+types::LE2ByteData
+    vpdEditor::get2BytesValue(BinaryVector::const_iterator iterator)
+{
+    types::LE2ByteData lowByte = *iterator;
+    types::LE2ByteData highByte = *(iterator + 1);
+    lowByte |= (highByte << 8);
+
+    return lowByte;
+}
+
+static void enableRebootGuard()
```

Comment:
Add a generic function in utils to make any DBus method call with or without parameters.

Reply:
TODO